includes/views/attachment-settings-hint.php

```diff
@@ -9,6 +9,6 @@
 	<p>
 		<?php /* translators: <a> tags */ ?>
 		<?php printf( wp_kses_post( __( 'It looks like you haven\'t setup any email attachments yet, check the settings under <b>%1$sDocuments > Invoice%2$s</b>', 'woocommerce-pdf-invoices-packing-slips' ) ), '<a href="'.$invoice_settings_url.'">', '</a>' ); ?>
-		<?php printf( '<a href="%s" style="display:block; margin-top:10px;">%s</a>', esc_url( add_query_arg( 'wpo_wcpdf_hide_attachments_hint', 'true' ) ), esc_html__( 'Hide this message', 'woocommerce-pdf-invoices-packing-slips' ) ); ?>
+		<?php printf( '<a href="%s" style="display:block; margin-top:10px;">%s</a>', esc_url( wp_nonce_url( add_query_arg( 'wpo_wcpdf_hide_attachments_hint', 'true' ), 'hide_attachments_hint_nonce', '_wpononce') ), esc_html__( 'Hide this message', 'woocommerce-pdf-invoices-packing-slips' ) ); ?>
```

Comment:
`_wpnonce` is not necessary, is already prefilled in the function arg. Btw you had a typo there: `_wpononce`

Reply:
I understand, thank you for clarifying that it was not necessary, since was already prefilled in the function arg.